app/src/main/java/com/github/wnder/GuessLocationActivity.java

```diff
@@ -300,8 +299,9 @@ private void showActualLocation() {
         scoreText.setText("Score: " + (int)score);
     }
 
-    private void openScoreBoardActivity() {
-        Intent intent = new Intent(this, ScoreBoardActivity.class);
+    private void openScoreboardActivity() {
+        Intent intent = new Intent(this, ScoreboardActivity.class);
+        intent.putExtra(ScoreboardActivity.EXTRA_PICTURE_ID, pictureID);
         startActivity(intent);
         finish();
```

Comment:
Maybe delete the finish(); so that we can come back to the guess after seeing the scoreboard

Reply:
Done!